### PR TITLE
Improve wording in Knife

### PR DESCRIPTION
**Pull request by tas50**
Each commit summary should sum up the change

### CODE REVIEW

**Comment by tas50** [comment]
The failure looks like a knife-windows issue actually.  I'll get a PR for that after this is merged

File: lib/chef/knife/bootstrap.rb [hunk 1]
```diff
@@ -189,7 +189,7 @@ class Bootstrap < Knife
 
       option :hint,
         :long => "--hint HINT_NAME[=HINT_FILE]",
-        :description => "Specify Ohai Hint to be set on the bootstrap target.  Use multiple --hint options to specify multiple hints.",
+        :description => "Specify Ohai Hint to be set on the bootstrap target. Use multiple --hint options to specify multiple hints.",
```
Comment (lamont-granquist) [inline lib/chef/knife/bootstrap.rb]: oh you're one of *those* people.

**Comment by lamont-granquist** [comment]
:+1:

**Comment by thommay** [comment]
:+1:

File: lib/chef/knife/bootstrap.rb (hunk 1 above)
Comment (thommay) [inline lib/chef/knife/bootstrap.rb]: Don't be ageist, lamont.

Comment (lamont-granquist) [inline lib/chef/knife/bootstrap.rb]: git off my lawn!!!

**Pull request merged**